### ISSUE TITLE
Check Request #27 - Component Upgrade Target Release Does Not Match Issue

### ISSUE THREAD

**Issue by rpelisse** [jboss-set/bug-clerk#42]
For instance, attaching a 7.0.x issue to a component upgrade targeted against 7.z

**Comment by rpelisse** [comment]
Just to have a concrete example: With issue https://issues.jboss.org/browse/JBEAP-6055 the target release is 7.0.Z.GA which matches the fix version 7.0.4.GA. 

A first crude implementation of this checks will use a simple comparaison of the begining of the both the target release and the fix version. In the previous example, if both fields does not start by "7.0", the inconsistency will be reported.

FYI, an issue that should be report would match the following query in JIRA:

```
project = JBEAP
AND resolution = Unresolved
AND description !~ Documentation
AND
    type != Documentation AND
    type = 'Component Upgrade'
AND
fixVersion = 7.0.4.GA AND
"Target Release"="7.1.0.GA"
ORDER BY affectedVersion, priority ASC
```

**Comment by rpelisse** [comment]
[Implemented](https://github.com/jboss-set/bug-clerk/commit/20c88233cf791708ef538b5d23a62a45213a8a35) and released with [Bugclerk 0.9.0.Final](https://github.com/jboss-set/bug-clerk/releases/tag/bugclerk-0.9.0.Final).
